### PR TITLE
Add provisioning script

### PR DESCRIPTION
**Pull request by clintonb**
ECOM-6565

### CODE REVIEW

**Comment by clintonb** [comment]
@rlucioni @davec-edx @jmbowman have a look

File: README.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
 # devstack
 
 Get up and running quickly with edX services.
+
+## Getting Started
+
+1. Run the provision command, if you haven't already, to configure the various services with super-users (for
+   development without the auth service) and tenants (for multi-tenancy).
+
+    ``` bash
```
Comment (rlucioni) [inline README.md]: You can remove the `bash` directive, it doesn't add anything in this case.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
 # devstack
 
 Get up and running quickly with edX services.
+
+## Getting Started
+
+1. Run the provision command, if you haven't already, to configure the various services with super-users (for
+   development without the auth service) and tenants (for multi-tenancy).
```
Comment (rlucioni) [inline README.md]: Mention the username and password used for these standalone superusers. As someone new to this devstack, I might not know to dig them out of `provision.sh`.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,18 @@
...
+
+    ``` bash
+    $ make devstack.provision
+    ```
+
+2. Start the services.
+
+    ```bash
+    $ make devstack.start
```
Comment (rlucioni) [inline README.md]: I'd also include a note about how to use `devstack.open` here. I might not know what to do when I run this and the containers are left running in the foreground.

**Review by rlucioni — APPROVED** [review]
Looks good. I made a few comments about the README, but documentation changes/additions can happen separately if that's easier/faster.

**Pull request merged**